src/components/layout/Sidebar.tsx

```diff
@@ -129,20 +130,24 @@ const Sidebar = ({
             >
               Genres
             </SidebarNavItem>
-            <SidebarNavItem
-              tabIndex={0}
-              eventKey="folders"
-              icon={<Icon icon="folder-open" />}
-              onSelect={handleSidebarSelect}
-              disabled={disableSidebar}
-              onKeyDown={(e: any) => {
-                if (e.key === ' ' || e.key === 'Enter') {
-                  history.push('/library/folder');
-                }
-              }}
-            >
-              Folders
-            </SidebarNavItem>
+            {useAppSelector((state) => state.config).serverType !== 'funkwhale' && (
```

Comment:
Just some stylistic changes:

Set the config as a variable at the top of the component with useAppSelector instead of using it inline.

```
const config = useAppSelector((state) => state.config)
```

Also import the `Server` enum from the `types` file and reference it:

```
config.serverType !== Server.funkwhale
```